setup.py

```diff
@@ -30,10 +30,14 @@ def read_version(package):
 ]
 
 swagger_ui_require = 'swagger-ui-bundle>=0.0.2,<0.1'
-flask_require = 'flask>=1.0.4,<3'
+flask_require = [
```

Comment:
Bit surprised that this doesn't give any errors when installing as line 48 adds `flask_require` as an element to a list.

Similarly, I would expect that line 103 also needs to be changed:
```
install_requires=install_requires + [flask_require],
```